pom.xml

```diff
@@ -225,6 +225,9 @@
 						<manifest>
 							<mainClass>org.reactome.addlinks.Main</mainClass>
 						</manifest>
+						<manifestEntries>
+							<Multi-Release>true</Multi-Release>
+						</manifestEntries>
```

Comment:
I'm not familiar with this tag - what does it do or convey?

Reply:
For Java versions > 8, this specifies a mulit-release jar:

https://in.relation.to/2017/02/13/building-multi-release-jars-with-maven/

https://stackoverflow.com/questions/60741008/maven-assembly-plugin-multi-releasetrue

I guess I needed it to get something running properly here. I can remove it, though if Justin's already run it on the Release server, it's unlikely to cause problems.

Reply:
Okay, just wondering